.github/workflows/continuous-integration.yml

```diff
@@ -54,50 +88,67 @@ jobs:
         path: rrtmgp-data
         ref: v1.8.1
     #
-    # Synchronize the package index
-    #
-    - name: Synchronize the package index
-      run: sudo apt-get update
-    #
-    # Install NetCDF-Fortran (compatible with all compilers)
-    #
-    - name: Install NetCDF-Fortran
-      run: sudo apt-get install libnetcdff-dev
-    #
     # Cache Conda packages
     #
     - name: Cache Conda packages
       uses: actions/cache@v4
       with:
         path: ~/conda_pkgs_dir
-        key: conda-pkgs
+        key:
+          ${{ runner.os }}-conda-${{ hashFiles('environment-ci.yml') }}
     #
     # Set up Conda
     #
     - name: Set up Conda
       uses: conda-incubator/setup-miniconda@v3
       with:
         miniforge-version: latest
-        activate-environment: rte_rrtmgp_test
-        environment-file: environment-noplots.yml
-        python-version: 3.11
+        activate-environment: rte_rrtmgp_ci
+        environment-file: environment-ci.yml
         auto-activate-base: false
+        use-mamba: true
         # Use the cache properly:
         use-only-tar-bz2: true
     #
+    # Install NetCDF-Fortran
+    #
+    - name: Install NetCDF-Fortran via Conda
+      if: runner.os == 'Linux'
+      run: conda install -y -c conda-forge netcdf-fortran
```

Comment:
What is the advantage of installing `netcdf-fortran` with Conda (given that it seems to be non-portable and we do that on Linux only)?

Reply:
It's actually quite portable, see their conda-forge [feedstock](https://anaconda.org/conda-forge/netcdf-fortran). All their installers across OSs and architectures are at v4.6.1. Since conda and conda-forge are great at managing such dependencies I would be inclined to use them as much as possible in this case.
The problem with Windows is that the `gfortran` version is rather old. You don't want to be testing with an old compiler but for packaging purposes it's fine.

Bigger picture: When you are packaging your feedstock one of the steps is to make sure that all your dependencies i.e. `netcdf-fortran` are available in conda-forge. There are ways around this (e.g. building from source), but the APT package manager is not one of them.